MAVProxy/mavproxy.py

```diff
@@ -908,10 +936,10 @@ def run_script(scriptfile):
         action='append',
         default=[],
         help='Load the specified module. Can be used multiple times, or with a comma separated list')
-    parser.add_option("--mav09", action='store_true', default=False, help="Use MAVLink protocol 0.9")
     parser.add_option("--mav10", action='store_true', default=False, help="Use MAVLink protocol 1.0")
-    parser.add_option("--mav20", action='store_true', default=True, help="Use MAVLink protocol 2.0")
+    parser.add_option("--mav20", action='store_true', default=False, help="Use MAVLink protocol 2.0")
```

Comment:
Default is auto-detect via mavversion == None. Should (in theory) pick up the correct mavlink version without interaction from the user.